apps/emqx_bridge/test/emqx_bridge_v2_api_SUITE.erl

```diff
@@ -1512,3 +1511,47 @@ t_raw_config_response_defaults(Config) ->
         )
     ),
     ok.
+
+t_older_version_nodes_in_cluster(matrix) ->
+    [
+        [cluster, actions],
+        [cluster, sources]
+    ];
+t_older_version_nodes_in_cluster(Config) ->
```

Comment:
Is this testcase going to be relevant in the future releases? If not, perhaps a manual one-off test is enough?

Reply:
I think this still might be relevant for any rolling upgrade scenario.